modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -89,6 +107,55 @@ For the EU control plane:
 # docker tag rtf-runtime-registry.kprod-eu.msap.io/mulesoft/rtf-agent:v1.12.0 <docker-server>/mulesoft/rtf-agent:v1.12.0 
 # docker push <docker-server>:5000/mulesoft/rtf-agent:v1.12.0
 ----
+--
+====
+
+
+== Configure Your Local Registry for Runtime Fabric
+
+To configure a local registry, you must gather and add the necessary credentials to synchronize the registries. 
+
+=== Before You Begin
+
+Ensure that you've performed the following tasks: 
+
+. Set up, configure, and test your private Docker image registry.
+. <<synchronize, Synchronize>> to your local registry all the Docker images you need to install Runtime Fabric:
++
+See the xref:release-notes::runtime-fabric/runtime-fabric-release-notes-2.x.x.adoc[Runtime Fabric release notes] for the required dependencies for your Runtime Fabric version. 
++
+[NOTE]
+Dependency versions are specific to the Runtime Fabric version. 
+
+To install Runtime Fabric with local registry using `rtfctl` or `helm`, select one of the following options:
```

Comment:
```suggestion
To install Runtime Fabric to the local registry using `rtfctl` or `helm`, select one of the following options:
```

Reply:
I am not sure if my comment makes sense. Are you installing to a local registry or by using a local registry?